src/components/HypothesisBuilder/components/HypothesisDetails/index.js

```diff
@@ -2,9 +2,9 @@ import React, { useCallback, useEffect, useRef, useState } from "react";
 import { debounce } from "lodash";
 import { useForm } from "react-hook-form";
 import { v4 as uuidv4 } from "uuid";
-import { Badge } from "../../../Badge";
+import cx from "clsx";
```

Comment:
don't rename it use `clsx`

Reply:
fixed: replace sx to slsx everywhere, remove dropdown from modal